pkg/cmd/pulumi/new.go

```diff
@@ -1083,6 +1090,21 @@ func templatesToOptionArrayAndMap(templates []workspace.Template,
 	return options, nameToTemplateMap
 }
 
+// formatDesc formats the template description nicely if the term width requires it to wrap around.
+func formatDesc(prefixWidth int, maxWidth int, desc string) string {
+	descFormatted := ""
+	descWidth := maxWidth - prefixWidth
+	for idx := descWidth; idx < len(desc); idx += descWidth {
+		buffer := prefixWidth
+		if strings.LastIndex(desc[:idx], " ") != -1 {
+			idx = strings.LastIndex(desc[:idx], " ")
+			buffer += (descWidth - idx - 1)
```

Comment:
I must be misunderstanding this, but won't idx be much larger than descWidth?
It's the index in the whole string, while descWidth is the width of a single line?

Reply:
Ah yeah, I think this only appeared to work initially because most template descriptions wrap around at most once... but definitely need to fix/test this 😄